snooty.toml

```
@@ -9,5 +9,6 @@ toc_landing_pages = [
 
 [constants]
 version = 4.3
+full-version = "{+version+}.2"
```

```suggestion
minor-version = 2
full-version = "{+version+}.{+minor-version+}"
```
Would it make sense to make the minor version a source constant? I like how readable this is (consistent level of abstraction throughout definition of the `full-version` constant).